geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/DestroyRegionCommand.java

```diff
@@ -77,4 +87,35 @@ public Result destroyRegion(
     return result;
   }
 
+  void checkForJDBCMapping(String regionPath) {
+    String regionName = regionPath;
+    if (regionPath.startsWith("/")) {
+      regionName = regionPath.substring(1);
+    }
+    InternalConfigurationPersistenceService ccService = getConfigurationPersistenceService();
+    if (ccService == null) {
+      return;
+    }
+    CacheConfig cacheConfig = ccService.getCacheConfig(null);
+    if (cacheConfig == null) {
+      return;
+    }
+    RegionConfig regionConfig = findRegionConfig(cacheConfig, regionName);
+    if (regionConfig == null) {
+      return;
+    }
+    Iterator<CacheElement> iterator = regionConfig.getCustomRegionElements().iterator();
```

Comment:
same here, you can use 
```suggestion
    RegionMapping mapping = CacheElement.findElement(cacheConfig.getCustomRegionElements(), "jdbc-mapping")
```

Reply:
will do